### DISCUSSION TITLE
Digital Humanities Arts and Social Sciences - how do we support people working in research software in this community?

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#126] [Discussion topics for #RSEAA24]
We have Dr. Arianna Ciula, Director, King’s Digital Lab, King’s College London as a keynote and a growing number of people working in research software in the Digital Humanities Arts and Social Science community.

There has been renewed interest in the UK about supporting Digital Humanities Arts and Social Sciences.

How can we support them?

**Comment by andrepiza** [comment]
The Alan Turing Institute, through the [AHRC*-funded Data/Culture project,](https://www.turing.ac.uk/research/research-projects/dataculture-building-sustainable-communities-around-arts-and-humanities) is working on a roadmap to enhance the UK’s computational research capability in Arts & Humanities (A&H). This initiative aims to address the challenges faced by researchers in conducting A&H data-driven research and will be one of the key outputs of the project informing AHRC’s plans. 
 
In recent years, many initiatives, including some funded by the AHRC, pointed to the lack of access to Research Software Engineering and Data Science expertise and infrastructure as one of the main challenges to creating a flourishing environment for computational research in A&H. Our work on the roadmap is bringing this issue to a focus. We believe that to be effective and deliverable, any proposition must be built with a wide community of stakeholders who will be some of the key deliverers and users of such enhanced capability. This work has started on July 30th with a diverse group of experts from across the UK and a final output should be submitted to the AHRC in the end of the year. 

We hope that this work will benefit researchers, professional associations, policy makers and others looking to find ways to remove barriers to digital research in A&H and create a thriving environment where finding or developing computational expertise and infrastructure is much more accessible.

*The [AHRC](https://www.ukri.org/councils/ahrc/) (Arts and Humanities Research Council) is part of the [UK Research and Innovation](https://www.ukri.org/) the UK's main state funder of research in Arts and Humanities
